src/styles/main.css

```diff
@@ -0,0 +1,188 @@
+body {
+  margin: 0;
+  font-family: Arial, Baghdad, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: 1px;
+  margin: 50px auto;
+}
+.calendar--month-length--28 .day:nth-child(n+29) {
+  display: none;
+}
+.calendar--month-length--29 .day:nth-child(n+30) {
+  display: none;
+}
+.calendar--month-length--30 .day:nth-child(n+31) {
+  display: none;
+}
+.calendar--month-length--31 .day:nth-child(n+32) {
+  display: none;
+}
+.calendar__start-day--mon > :first-child {
+  margin-left: 0px;
+}
+.calendar__start-day--tue > :first-child {
+  margin-left: 101px;
+}
+.calendar__start-day--wed > :first-child {
+  margin-left: 202px;
+}
+.calendar__start-day--thu > :first-child {
+  margin-left: 303px;
+}
+.calendar__start-day--fri > :first-child {
+  margin-left: 404px;
+}
+.calendar__start-day--sat > :first-child {
+  margin-left: 505px;
+}
+.calendar__start-day--sun > :first-child {
+  margin-left: 606px;
+}
+
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100px;
+  height: 100px;
```

Comment:
The grey color for the square is not specified. It is better to use a specific grey color code, for example: #808080.